components/coachmark/index.css

```diff
@@ -10,127 +10,198 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+	font-weight: var(--mod-coachmark-content-font-weight, var(--spectrum-coachmark-content-font-weight));
+	font-family: var(--mod-coachmark-content-font-family, var(--spectrum-coachmark-content-font-family));
+	font-style: var(--mod-coachmark-content-font-style, var(--spectrum-coachmark-content-font-style));
+	line-height: var(--mod-coachmark-content-line-height, var(--spectrum-coachmark-content-line-height));
 }
 
-.spectrum-CoachMarkIndicator--quiet {
-  min-inline-size: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 2.75);
-  min-block-size: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 2.75);
+.spectrum-CoachMark-footer {
+	display: grid;
+	align-items: end;
+	margin-block-start: 0;
+	padding-block-end: var(--mod-coachmark-padding, var(--spectrum-coachmark-padding));
 
-  .spectrum-CoachMarkIndicator-ring {
-    inset-block-start: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 0.75);
-    inset-inline-start: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 0.75);
+	.spectrum-ButtonGroup {
+		grid-column-start: 2;
+	}
+}
 
-    inline-size: var(--spectrum-coachmark-quiet-indicator-ring-diameter);
-    block-size: var(--spectrum-coachmark-quiet-indicator-ring-diameter);
-    animation: pulse--quiet var(--spectrum-coachmark-animation-indicator-ring-duration) linear infinite;
+.spectrum-CoachMark-title {
+	color: var(--mod-coachmark-title-color, var(--spectrum-coachmark-title-color));
+	font-size: var(--mod-coachmark-title-font-size, var(--spectrum-coachmark-title-font-size));
+	font-weight: var(--mod-coachmark-title-text-font-weight, var(--spectrum-coachmark-title-text-font-weight));
+	font-family: var(--mod-coachmark-title-font-family, var(--spectrum-coachmark-title-font-family));
+	font-style: var(--mod-coachmark-title-font-style, var(--spectrum-coachmark-title-font-style));
+	line-height: var(--mod-coachmark-title-text-line-height, var(--spectrum-coachmark-title-text-line-height));
+	margin-block-end: 0;
+}
 
-    &:nth-child(1) {
-      animation-delay: calc(var(--spectrum-coachmark-animation-indicator-ring-duration) * var(--spectrum-coachmark-quiet-animation-indicator-ring-inner-delay-multiple));
-    }
-  }
+.spectrum-CoachMark-step {
+	justify-self: start;
+	color: var(--mod-coachmark-step-color, var(--spectrum-coachmark-step-color));
+	font-size: var(--mod-coachmark-step-font-size, var(--spectrum-coachmark-step-font-size));
+	font-weight: var(--mod-coachmark-step-text-font-weight, var(--spectrum-coachmark-step-text-font-weight));
+	font-style: var(--mod-coachmark-step-font-style, var(--spectrum-coachmark-step-font-style));
+	line-height: var(--mod-coachmark-step-text-line-height, var(--spectrum-coachmark-step-text-line-height));
+	white-space: nowrap;
+	margin-block-end: calc(var(--mod-coachmark-step-step-to-bottom, var(--spectrum-coachmark-step-step-to-bottom)) - var(--mod-coachmark-padding, var(--spectrum-coachmark-padding)));
```

Comment:
Could/should this token and mod avoid the repeated "step" in the name? `...coachmark-step-to-bottom`?